common/app/experiments/Experiments.scala

```diff
@@ -37,3 +37,12 @@ object OfferHttp3
       sellByDate = LocalDate.of(2022, 12, 6),
       participationGroup = Perc0B,
     )
+
+object KeyEventsCarousel
+    extends Experiment(
+      name = "key-events-carousel",
+      description = "Displays key events carousel on live blogs and hides old key events container",
```

Comment:
```suggestion
      description = "When ON, live blog key events are displayed in a carousel",
```

I find the 'When ON, ..." convention a real aid to comprehension, especially when skimming these switches.

Reply:
Looks great, thanks for suggesting Oliver!